BetonQuest-core/src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -525,12 +528,7 @@ public static void playSound(String playerID, String soundName) {
      * @return the default package, as specified in the config
      */
     public static ConfigPackage getDefaultPackage() {
-        String name = plugin.getConfig().getString("default_package");
-        ConfigPackage pack = getPackages().get(name);
-        if (pack == null) {
-            pack = getPackages().get("default");
-        }
-        return pack;
+        return getPackages().get(defauptPackage);
```

Comment:
The value is only set when Config is instantiated but is accessed here in a static function. Is there any risk that this gets called before Config exists?

Reply:
I'm going to reply to myself here. Looks like getPackages() also uses a local variable so it's no more risky so probably can be assumed fine. Might be worth just giving it an initial value of default just to be safe.